README.md

```diff
@@ -101,6 +101,58 @@ add `priority` label when you start backing us :smile:
 
 See: [docs/deployment.md](docs/deployment.md)
 
+# Development
+
+> This short guide is meant to give a guide as to how github apps are made and basic steps to reproduce the OpenCollective Bot should you wish to make futher changes.  
+
+ ## Installation
+
+Feel free to clone or download this project using the **git clone** and install using **Yarn install** or **Npm install** to install required dependencies
```

Comment:
This project use `yarn`, not `npm`. We should not mention `npm`.

Reply:
Alright.  
Noted